src/Swift.Bindings/src/Demangler/Swift5Reducer.cs

```diff
@@ -34,14 +35,17 @@ List<MatchRule> BuildMatchRules () =>
             Name = "Global", NodeKind = NodeKind.Global, Reducer = ConvertFirstChild
         },
         new MatchRule() {
-            Name = "ProtocolWitnessTable", NodeKind = NodeKind.ProtocolWitnessTable, Reducer = ConvertProtocolWitnessTable
+            Name = "ProtocolConformance", NodeKindList = new List<NodeKind>() { NodeKind.ProtocolWitnessTable, NodeKind.ProtocolConformanceDescriptor, } , Reducer = ConvertProtocolWitnessTable
```

Comment:
Wouldn't be better to rename the `ConvertProtocolWitnessTable` to something like `ConvertProtocolConformance` since now we use it for more than just witness tables?